src/System/Taffybar/Widget/NetworkGraph.hs

```diff
@@ -20,4 +25,9 @@ networkGraphNew config interfaces = do
       toLogScale = logScale 2 (2 ** 32)
       toSample (up, down) = map (toLogScale . fromRational) [up, down]
       sampleBuilder = return . toSample . getUpDown
-  channelGraphNew config chan sampleBuilder
+  widget <- channelGraphNew config chan sampleBuilder
+  void $ channelWidgetNew widget chan $ \speedInfo ->
+    let (up, down) = sumSpeeds $ map snd speedInfo
+        tooltip = showInfo defaultNetFormat 3 (fromRational down, fromRational up)
```

Comment:
shouldn't we let this format be configurable?

Reply:
Yes, but for this we need to change the signature of `networkGraphNew`. Is it ok? Or probably create a new function.

Reply:
okay this confused me for a very long time when reading this:

When you name something like this tooltip, it makes it seem like it is some type of widget. The name should make it clear that this is just a string

Reply:
> Yes, but for this we need to change the signature of `networkGraphNew`. Is it ok? Or probably create a new function.

Right just add a parameter and rename this function. Then make a new function with the same name as this one and provide the default configuration calling the function with the added parameter.

Reply:
Added new function with config.